fastcli/fastcli.py

```diff
@@ -108,14 +108,14 @@ async def main(
 
     results = [task.result() for task in done | pending]
     mb = sum(results) * 8 / 1024 / 1024
-    return mb / duration
+    return mb / duration, resp_json
 
 
-def run(*, timeout: float = 30, verbosity: int = logging.WARNING) -> float:
+def run(*, timeout: float = 30, verbose: int = logging.WARNING) -> float:
```

Comment:
I generally prefer `verbose` when it's a boolean, and `verbosity` when an integer (or accumulator).

Reply:
Ahh, I see you were fixing an error I made here. Sorry, and thanks!